Application/People/Meta/Student/Service/Service/Student.php

```diff
@@ -82,7 +84,9 @@ public function insertStudent(
         $tblStudentLocker = null,
         $tblStudentBaptism = null,
         $tblStudentIntegration = null,
-        $SchoolAttendanceStartDate = ''
+        $SchoolAttendanceStartDate = '',
```

Comment:
Codacy found an issue: [Incorrect spacing between argument "$SchoolAttendanceStartDate" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)